bundles/org.openhab.io.neeo/src/main/resources/OH-INF/i18n/neeo.properties

```diff
@@ -0,0 +1,8 @@
+io.config.neeo.checkStatusInterval.label = Check Status Interval (seconds)
+io.config.neeo.checkStatusInterval.description = The interval (in seconds) to check the status of the brain
+io.config.neeo.exposeAll.label = Expose All
+io.config.neeo.exposeAll.description = Expose all things
+io.config.neeo.exposeNeeoBinding.label = Expose NEEO Binding
+io.config.neeo.exposeNeeoBinding.description = Expose things found by the NEEO Binding
+io.config.neeo.searchLimit.label = Search Limit
+io.config.neeo.searchLimit.description = The maximum number of results to return for a search
```

Comment:
We don't have an entry for the binding label

Reply:
I have made a fix for it @lolodomo!